imap_processing/hit/packet_definitions/P_HIT_SCIENCE.xml

```diff
@@ -0,0 +1,371 @@
+			<xtce:Parameter name="NADCSTIM" parameterTypeRef="UINT16" shortDescription="adc-stim events">
+				<xtce:LongDescription>ADC-calibration STIM events</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NODD" parameterTypeRef="UINT16" shortDescription="odd events">
+				<xtce:LongDescription>odd events</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NODDFIX" parameterTypeRef="UINT16" shortDescription="odd events that were fixed in sw">
+				<xtce:LongDescription>fixed odd events</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NMULTI" parameterTypeRef="UINT16" shortDescription="events with multiple hits in a single detector (may be crosstalk)">
+				<xtce:LongDescription>events with multiple hits in relevant layers</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NMULTIFIX" parameterTypeRef="UINT16" shortDescription="multi events that were fixed in sw">
+				<xtce:LongDescription>fixed multi events</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NBADTRAJ" parameterTypeRef="UINT16" shortDescription="bad trajectory">
+				<xtce:LongDescription>events rejected for inconsistent/bad trajectory</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NL2" parameterTypeRef="UINT16" shortDescription="events sorted into L12 event category">
+				<xtce:LongDescription>events sorted into L12 event category</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NL3" parameterTypeRef="UINT16" shortDescription="events sorted into L123 event category">
+				<xtce:LongDescription>events sorted into L123 event category</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NL4" parameterTypeRef="UINT16" shortDescription="events sorted into L1423 event category">
+				<xtce:LongDescription>events sorted into L1423 event category</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NPEN" parameterTypeRef="UINT16" shortDescription="events sorted into PEN (penetrating) event category">
+				<xtce:LongDescription>events sorted into PEN (penetrating) event category</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NFORMAT" parameterTypeRef="UINT16" shortDescription="nothing currently goes in this slot">
+				<xtce:LongDescription>events handled b the telemetry event formatter</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NASIDE" parameterTypeRef="UINT16" shortDescription="A-side events">
+				<xtce:LongDescription>events the software assumed were A-side events</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NBSIDE" parameterTypeRef="UINT16" shortDescription="B-side events">
+				<xtce:LongDescription>events the software assumed were B-side events</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NERROR" parameterTypeRef="UINT16" shortDescription="events that caused a processing error - should never happen">
```

Comment:
I don't know if this is a generator problem or not, but it really bothers me that this file mixes parameter definitions for short and long descriptions. I'd prefer to see: 

```
<xtce:ShortDescription>events the software assumed were B-side events</xtce:ShortDescription>
<xtce:LongDescription>events the software assumed were B-side events</xtce:LongDescription>
```

Reply:
Good catch. The [HIT housekeeping xml](https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/dev/imap_processing/hit/packet_definitions/P_HIT_HSKP.xml) uses the structure you prefer. @sdhoyt, do you recall running into any formatting issues with the short and long descriptions when generating the housekeeping xml?

Reply:
It looks like the telemetry_generator.py code is setup to write the short description as a parameter attribute and the long description as a sub element.

https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/440ed182c7fd269e026275f8c6b2fd5a66e988fd/tools/xtce_generation/telemetry_generator.py#L271-L276

I could update this part of the code as follows to get both descriptions as sub elements. Thoughts?

```python
            # Add descriptions if they exist
            if pd.notna(row.get("shortDescription")):
                description = Et.SubElement(parameter, "xtce:ShortDescription")
                description.text = row.get("shortDescription")
            if pd.notna(row.get("longDescription")):
                description = Et.SubElement(parameter, "xtce:LongDescription")
                description.text = row.get("longDescription")
```

Reply:
If you wouldn't mind, I would really prefer them to match. Seems like it's a really easy change

Reply:
Oh, it looks like the generator was setup the way I suggested above and was changed in March https://github.com/IMAP-Science-Operations-Center/imap_processing/pull/364

@tech3371 Do you recall the motivation for this update? I don't want to change it back if there was need for this structure in the xml files.

Reply:
This update was made because the way we were doing before was wrong according to XTCE document. Short description should be in the parameter tab. This was updated when Gavin was implementing feature of getting short and long description from unpacked data. He spotted this bug.

Reply:
We all will need to update our XTCE definition if we plan to use short and long description from unpacked data. It shouldn't break unpacking data if we leave it as it is. We won't see short description I believe.

Reply:
This is helpful background, thanks for sharing! I will leave telemetry_generator.py as is